studio/components/ui/ConsentToast.tsx

```diff
@@ -0,0 +1,36 @@
+import { noop } from 'lodash'
+import Link from 'next/link'
+import { Button } from 'ui'
+
+interface ConsentToastProps {
+  onAccept: () => void
+  onOptOut: () => void
+}
+
+const ConsentToast = ({ onAccept = noop, onOptOut = noop }: ConsentToastProps) => {
+  return (
+    <div className="space-y-3">
+      <div className="text-sm">
+        <p>
+          This site <span className="underline font-medium">does not</span> use cookies 🍪
+        </p>
+        <p>We use friendly server side analytics to measure and improve your experience</p>
+      </div>
+      <div className="flex items-center space-x-2">
+        <Button type="default" onClick={onAccept}>
+          Accept
+        </Button>
+        <Button type="text" onClick={onOptOut}>
+          Opt out
+        </Button>
+        <Link href="/">
```

Comment:
Should this go somewhere?

Reply:
yup this is still WIP - need a legal page from Pam

Reply:
To unblock this, you can use this https://supabase.com/privacy temporarily @joshenlim but we do plan to update the wording there so we are not deploying this until that's sorted